lib/active_model/serializer/caching.rb

```diff
@@ -89,6 +89,10 @@ def fragmented(serializer)
         # https://github.com/rails-api/active_model_serializers/pull/1249#issuecomment-146567837
         def cache(options = {})
           self._cache = ActiveModelSerializers.config.cache_store if ActiveModelSerializers.config.perform_caching
```

Comment:
Older versions of AMS used a method `perform_caching` named same as rails.  I think that might be a good idea